src/controller/BUILD.gn

```diff
@@ -18,6 +18,12 @@ import("${chip_root}/src/lib/lib.gni")
 import("${chip_root}/src/platform/device.gni")
 import("${chip_root}/src/platform/python.gni")
 
+source_set("nodatamodel") {
```

Comment:
Self-comment to prevent auto-submit:

Still considering removing this or using this. One of the two must happen before this PR is final.

Reply:
It seems Matter.framework requires emptydatamodel

Reply:
also this name is hard to read... maybe `no-data-model` or `no-ember-data-model` or similar is easier to read. I was originally hoping that the "just remove the file" is the right path and had no good name.